test/testhelper/settings/common.go

```diff
@@ -55,7 +56,7 @@ const (
 	BAREMETAL         = "BAREMETAL"
 	AliCloud          = "ALI_CLOUD"
 	CONTAINER         = "CONTAINER"
-	DefaultImage      = "registry.cn-qingdao.aliyuncs.com/sealer-io/kubernetes:v1.19.8"
+	DefaultImage      = "registry.cn-qingdao.aliyuncs.com/sealer-io/kubernetes:v1.22.15"
```

Comment:
update defaultImage to `docker.io/sealerio/kubernetes:v1.22.15`

Reply:
It's the same everywhere else

Reply:
Already updated.